SpatialGDK/Source/SpatialGDK/Public/Utils/InterestFactory.h

```diff
@@ -25,7 +25,7 @@ class SPATIALGDK_API InterestFactory
 	Worker_ComponentData CreateInterestData() const;
 	Worker_ComponentUpdate CreateInterestUpdate() const;
 
-	static Interest CreateServerWorkerInterest();
+	static Interest CreateServerWorkerInterest(const USpatialNetDriver& NetDriver);
```

Comment:
as mentioned elsewhere I think, can just pass in LB strat here?

Reply:
done